### ISSUE TITLE
Article: .gitignore

### ISSUE THREAD

**Issue by prokopsimek** [DXHeroes/knowledge-base#90]

**Comment by zeman-j4** [comment]
What this is about: Having a well set gitignore file will keep your codebase clean from all the files you do not want to share nor need to be shared - log files, dependencies, private data and so on. Set it up to stay safe and thoughtful.

You can start your research here: https://www.pluralsight.com/guides/how-to-use-gitignore-file
https://docs.github.com/en/github/using-git/ignoring-files

Contribution guide: https://github.com/DXHeroes/knowledge-base-content/blob/master/CONTRIBUTING.md

**Comment by ridlees** [comment]
Add short template for Java, Javascript, Typescript

**Comment by adithyaakrishna** [comment]
@prokopsimek  Would love to work on this :)

**Comment by ridlees** [comment]
@adithyaakrishna Hello, sure, go for it!

**Comment by DominikMilaniak** [comment]
Hi @adithyaakrishna! I am extremely happy you chose our project! 😄 I see you are assigned to "User journey" article as well - will you be able to produce 2 articles? If yes, I will happily assign you to this article too :)

**Comment by adithyaakrishna** [comment]
Yes @DominikMilaniak I'll be able to, but it may take a day or two from both of the articles, hope its okay 😊

**Comment by DominikMilaniak** [comment]
That is amazing @adithyaakrishna! Of course it is okay, take as much time as you need :) I'm assigning you to this as well. Looking forward to your articles! ❤️

**Comment by DominikMilaniak** [comment]
**Hi again** @adithyaakrishna! 🙂 

I see it is a few days since our last contact, how is it going so far with the article? Is anything not clear enough or do you need help with something? I am in no way pushing you, you can take as much time as you need 🙂  Just wanted to get in touch to make sure everything is ok 👍

**Comment by adithyaakrishna** [comment]
Hey @DominikMilaniak , Yes Just started on this. Due to some personal work wasn't able to commit to this. But will update as soon as I'm done :)
